.pre-commit-config.yaml

```
@@ -16,7 +16,7 @@ repos:
     hooks:
       - id: black
   - repo: https://github.com/crate-ci/typos
-    rev: v1.30.2
+    rev: v1
```

```suggestion
    rev: v1.30.2
```

`v1` and `v1.30.2` are identical, but the former seems to be an accidentally commited tag.